pkg/registry/core/rest/storage_core.go

```diff
@@ -218,7 +218,7 @@ func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(restOptionsGetter generi
 	if utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) && c.SecondaryServiceIPRange.IP != nil {
 		var secondaryServiceClusterIPRegistry rangeallocation.RangeRegistry
 		secondaryServiceClusterIPAllocator, err = ipallocator.NewAllocatorCIDRRange(&c.SecondaryServiceIPRange, func(max int, rangeSpec string) (allocator.Interface, error) {
-			mem := allocator.NewAllocationMap(max, rangeSpec)
+			mem := allocator.NewAllocationMapReserved(max, rangeSpec)
```

Comment:
Do you also need to change the call in https://github.com/openshift/kubernetes/blob/a4094862b8876018b7385008087c96497e4fd149/pkg/registry/core/rest/storage_core.go#L238 ?

Reply:
that is for nodePorts, they share the same allocator implementation, but we only care about IPs

Reply:
I guess not.